Changes

```diff
@@ -226,7 +226,7 @@ v1.1.0 2017-04-04 Public pre-release
 
 1.0.1    2015-04-07
  - State clearly that a given name is not a domain when it is not a domain
- - Send "Net::LDNS not compiled with libidn" to STDERR, not STDOUT
```

Comment:
That should not be changed.

Reply:
Oups ^^'

Reply:
fixed (hopefully I haven't done the same mistake in the other repos)